integration-tests/mockApis/elite2api.js

```diff
@@ -102,6 +102,34 @@ module.exports = {
       },
     })
   },
+  stubPrison: prisonId => {
+    const getPrisonDescription = () => {
+      const prisons = [
+        { id: 'LEI', description: 'Leeds' },
+        { id: 'MDI', description: 'Moorland' },
+      ]
+      return prisons.find(prison => prison.id === prisonId).description
+    }
```

Comment:
interesting, could remove the find and the method call by just defining descriptions as data structure: 
```js
const descriptions = {
  LEI: 'Leeds',
  MDI: 'Moorland'
}
...
 jsonBody: {
          agencyId: prisonId,
          description: description[agencyId],
          agencyType: 'INST',
          active: true,
        },
```

Reply:
yes, simpler ...
done